### PR TITLE
Fullscreen search & Navbar change

### PR DESCRIPTION
**Pull request by kenos1**
better ui pretty much

### CODE REVIEW

File: components/interactive/SearchBar.tsx
Comment (Compositr) [inline components/interactive/SearchBar.tsx:44]: Only thing left until approve is making this button look more clickable. I suck at UI soooo @kenos1

**Comment by kenos1** [comment]
i think we are ready

**Review by Compositr — APPROVED** [review]

**Comment by Compositr** [comment]
lgtm, merge when ready

**Pull request merged**
